apps/my_new_app/management/commands/generate_data.py

```diff
@@ -32,6 +35,95 @@ def __generate_name(self,
                         inc: int) -> str:
         return f'{name} {inc}'
 
+    def _generate_users(self,user_number: int,initial_time) -> None:
+        """Generate user objects"""
+        
+        REQUIRED_SUPERUSER_QUANTITY = 1
+        _email_patterns: tuple = (
+            'gmail.com','outlook.com','yahoo.com',
+            'inbox.ru','inbox.ua','inbox.kz',
+            'yandex.ru','yandex.ua','yandex.kz',
+            'mail.ru','mail.ua','mail.kz'
+        )
+        
+        def get_username(first_name: str, last_name: str) -> str:
+            username: str = first_name.lower() + '_' + last_name.lower()
+            return username
+        
+        def get_email(first_name: str, last_name: str) -> str:
+            email_identification: str = random.choice(_email_patterns)
+            email: str = first_name.lower() + '.' + last_name.lower() + \
+                '@' +email_identification
+            return email
+        
+        def get_password() -> str:
+            _passwd_pattern: str = 'abcde12345'
+            _passwd_max_length: int = 8
+            _password_min_length: int = 1
```

Comment:
Django requires min password length of 8, so its better to declare `_passwd_max_length: int = 8` as `_passwd_length: int = 8`